src/stars--1.css

```
@@ -0,0 +1,11 @@
+.stars--1 .stars__star {
+  background-image: url(../src/images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  margin: 2px;
```

```suggestion
  margin-right: 4px;
```

There are no top and bottom margins in the design, also don't forget to remove margin-right from the last stars in a row